server/chat.ts

```diff
@@ -1844,23 +1853,14 @@ export const Chat = new class {
 				this.loadPluginDirectory(path, depth);
 			} else {
 				try {
-					this.loadPlugin(path);
+					this.loadPluginData(require(path), path.split('/').pop()?.split('.').shift() || path);
```

Comment:
If the plugin file is `asdf.jkl.ts`, the plugin name should be `asdf.jkl`, not `asdf`.